website/docs/add-secure-apps/providers/rac/rac_credentials_prompt.md

```diff
@@ -0,0 +1,80 @@
+---
+title: RAC Credentials Prompt
+---
+
+## About RAC Credentials Prompt
+
+The RAC provider can be configured to prompt users for their credentials when connecting to endpoints. This is particulalry useful for establishing RDP connections to modern Windows systems that often require credentials to establish a connection.
+
+After implementing this configuration, when connecting to an RAC endpoint users will be prompted to enter their credentials which will then be passed to the RAC endpoint. This means that static credentials do not need to be set in the RAC provider, property mapping, or endpoint.
+
+This configurations requires:
+
+1. Creating an authorization flow.
```

Comment:
Is this the exact order the steps need to be done in? If the order of the steps don't matter, use bullets.